docs: Add more documentation (#18)
aneoconsulting/ArmoniK.Extensions.Cpp: .docs/content/2.tests.md

# Run tests

## How to use the ArmoniK C++ SDK

### Build worker and dynamic worker (Linux only)

The ArmoniK dynamic worker image must be built to use ArmoniK with the C++ SDK. This worker image is then used in the ArmoniK deployment phase. The worker needs a shared library containing the implementation of the mandatory symbols that allow  ArmoniK to launch the required services.

In order to build the dynamic worker image (```armonik-sdk-cpp-dynamicworker```) and the worker test (```libArmoniK.SDK.Worker.Test.so```), one should run the following script:
- ```build-deploy-end2end.sh```

This script generates the dynamic worker image that is used to deploy ArmoniK with. The script also generates the shared library used by the worker to perform the ArmoniK call on the desired services.  The script will also copy this shared library into the right folder so it can be loaded by the ArmoniK worker.

>  One can also use the worker image available in docker hub ANEOand build the shared library separately.

Reviewer: ```suggestion
 It is also possible to use the worker image available in the [docker hub ANEO](https://hub.docker.com/r/dockerhubaneo/armonik-sdk-cpp-dynamicworker) and build the shared library separately.
```
